xbmc/guilib/GUIDialog.cpp

```diff
@@ -206,15 +195,21 @@ void CGUIDialog::Open_Internal(bool bProcessRenderLoop, const std::string &param
 }
 
 void CGUIDialog::Open(const std::string &param /* = "" */)
+{
+  CGUIDialog::Open(m_modalityType != DialogModalityType::MODELESS, param);
+}
+
+
+void CGUIDialog::Open(bool bProcessRenderLoop, const std::string &param /* = "" */)
```

Comment:
> Left-align `*` and `&` to the base type they modify.

https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#52-pointer-and-reference-types